Example/ZSWTappableLabel/3D Touch/3DTouchSwiftViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  3DTouchSwiftViewController.swift
+//  ZSWTappableLabel_Example
+//
+//  Created by Zac West on 4/20/19.
+//  Copyright © 2019 Zachary West. All rights reserved.
+//
+
+import UIKit
+import ZSWTappableLabel
+import SafariServices
+
+class DDDTouchSwiftViewController: UIViewController, ZSWTappableLabelTapDelegate, ZSWTappableLabelLongPressDelegate, UIViewControllerPreviewingDelegate {
```

Comment:
3d works in swift, remember 📷ViewController ?

Reply:
Hm, I think if it worked before it doesn't now. https://repl.it/repls/PoorEmotionalAutotote

> main.swift:1:7: error: class name can only start with a letter or underscore, not a number